dev/sparktestsupport/modules.py

```diff
@@ -1040,7 +1040,14 @@ def __hash__(self):
 pyspark_errors = Module(
     name="pyspark-errors",
     dependencies=[],
-    source_file_regexes=["python/pyspark/errors"],
+    source_file_regexes=[
+        # [SPARK-44544] Force the execution of pyspark_errors when there is any changes
+        # in PySpark, since the Python Packaging Tests is only enabled within this module.
+        # This module is the smallest Python test modules, it only contains 1 test file
+        # and normally take < 2 seconds, so the additional cost is small.
+        "python/",
```

Comment:
cc @HyukjinKwon I find it is pretty hard to me to decouple the packaging tests from other pyspark tests. what about this workaround?

Reply:
SGTM